### ISSUE TITLE
Using @swc-jotai/react-refresh above version 0.0.7 isn't working with @swc/core for version above 1.3.40

### ISSUE THREAD

**Issue by abouteiller-niji** [pmndrs/swc-jotai#16]
Hello, I was updating dependencies of my project until i got blocked with theses two differents versions.

I tried to update both libs versions separately but I found that they couldn't work together starting from theses versions:

"@swc-jotai/react-refresh": "0.0.7"
"@swc/core": "1.3.40"

I've also seen that this project was only using the version 0.79.56 of @swc/core in the last version.
Do I need to downgrade swc version to match this one or you think that this can be fixed without it ?

If you want any other information, I can give you them.

**Comment by yf-yang** [comment]
I am using @swc-jotai/react-refresh 0.10.0 and next 13.4.19 (it has swc-core 1.3.55). It seems @swc-jotai/debug-label is not working. I tried to add the following to my next.config.js
``` javascript
experimental: {
    swcPlugins: [
      ['@swc-jotai/debug-label', {}],
      ['@swc-jotai/react-refresh', {}],
    ],
  },
```
However in react devtools
<img width="473" alt="image" src="https://github.com/pmndrs/swc-jotai/assets/36890796/9afad054-f7ba-43c4-9fcf-bcc00406c765">
And I checked the bundled code in my .next folder, the '.debugLabel' code cannot be found.

I am not sure if this is the same issue.
